packages/celo-typescript/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "composite": true,
+    "declaration": true,
+    "downlevelIteration": true,
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "jsx": "react",
+    "lib": ["DOM",  "DOM.Iterable", "es2015", "es2016"],
```

Comment:
Should this maybe be up to each project? Tbh I don't know the impact of leaving it in, so might be fine, but removing it would make it more generic.